manifest.json

```diff
@@ -20,7 +20,11 @@
 	{
 		"128" : "icons/emacs.png"
 	},
-	"options_page": "fancy-settings/source/index.html",
+	"options_ui": {
+		"page": "fancy-settings/source/index.html",
+		"browser_style": true,
```

Comment:
I see only `browser_style` is not supported which is not really needed anyway. https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/options_ui#Browser_compatibility